### ISSUE TITLE
Various additional Automator improvements

### ISSUE THREAD

**Issue by cyip92** [IvarK/AntimatterDimensionsSourceCode#2133]
There's some additional functionality that's been mentioned in various places which haven't yet been put "to paper" in an issue yet, so here they are.

**Block Automator functionality**
The block automator has some serious lingering and still-unfixed issues, see #1837 although there may potentially be more to find than just what's described in that issue once someone dives into that part of the code again.

**Switching between Block and Text**
It was mentioned a while ago (I believe by omsi) that for the sake of approachability, block mode should probably be the default and/or that switching between modes should be much more visually obvious, and I think that's a good idea. Also, an additional suggestion from Buck and Giri:
![image](https://user-images.githubusercontent.com/4493131/148807960-67dd4cbf-6367-4342-8ecf-1a828973f1c9.png)
(This of course requires that block is less buggy than it currently is; we probably don't want to showcase block mode if it's buggier than text mode)

**Command categorization**
More from a conversation between Buck and Giri which I think is worth considering here too:
![image](https://user-images.githubusercontent.com/4493131/148808846-887e1b5c-a4c3-4d39-9e6e-ef1ba5d3a4c5.png)
Visually, blockmato mode might also benefit from some additional "syntax highlighting"
![image](https://user-images.githubusercontent.com/4493131/148808537-6f449e5d-9115-4969-b071-943088aff433.png)
